website/docs/r/org_vdc.html.markdown

```diff
@@ -144,8 +144,42 @@ resource "vcd_org_vdc" "my-vdc" {
   description = "The pride of my work"
   org         = "my-org"
   # ...  
-  default_vm_sizing_policy_id = vcd_vm_sizing_policy.size_1.id
-  vm_sizing_policy_ids        = [vcd_vm_sizing_policy.size_1.id, vcd_vm_sizing_policy.size_2.id]
+  default_compute_policy_id = vcd_vm_sizing_policy.size_1.id
+  vm_sizing_policy_ids      = [vcd_vm_sizing_policy.size_1.id, vcd_vm_sizing_policy.size_2.id]
+}
+```
+
+## Example Usage (With VM Placement Policies)
+
+```hcl
+data "vcd_provider_vdc" "pvdc" {
+  name = "my-pvdc"
+}
+
+# This VM group needs to exist in the backing vSphere
+data "vcd_vm_group" "vmgroup" {
+  name            = "vmware-licensed-vms"
+  provider_vdc_id = data.vcd_provider_vdc.pvdc.id
+}
+
+resource "vcd_vm_placement_policy" "new-placement-policy" {
+  name            = "place-in-vmware-licensed"
+  provider_vdc_id = data.vcd_provider_vdc.pvdc.id
+  vm_group_ids    = [data.vcd_vm_group.vmgroup.id]
+}
+
+data "vcd_vm_placement_policy" "existing-policy" {
+  name            = "place-in-company-licensed"
+  provider_vdc_id = data.vcd_provider_vdc.pvdc.provider_vdc_id
```

Comment:
You know, I can't find this `provider_vdc_id` attribute in the docs. Maybe I'm looking in the wrong place. BTW, how is it different from the normal `id`?

Reply:
Actually it's the attribute mentioned in `website/docs/d/vm_placement_policy.html.markdown` (from VM Placement Policies).

As opposed to VM Sizing Policies, which are unequivocally fetched by their name (due that VCD enforces uniqueness in the names), VM Placement Policies can have the **same name** if they belong to different pVDCs.

In other words, to unequivocally fetch a VM Placement Policy, you need the tuple **name+pVDC** (and here we use the pVDC ID to identify the pVDC)

Reply:
Sorry, I was not clear enough. What I can't find is the `data.vcd_provider_vdc.pvdc.provider_vdc_id` in docs. I'm looking for it here:
https://github.com/vmware/terraform-provider-vcd/pull/904/files#diff-b7e0d433b971754aa439fe03ccaf5a581c1f22a86fc6f8354f49293be038854dR35-R53

Reply:
Ah, it was a typo indeed. Fixed!

Reply:
Yay! :) Thanks!